ivy_tests/test_ivy/test_frontends/test_numpy/test_fft/test_discrete_fourier_transform.py

```diff
@@ -189,3 +189,35 @@ def test_numpy_rfftfreq(n, sample_rate, frontend, test_flags, fn_tree, on_device
         n=n,
         d=d,
     )
+
+
+@handle_frontend_test(
+    fn_tree="numpy.fft.rfft2",
+    dtype_input_axis=helpers.dtype_values_axis(
+        available_dtypes=None,
+        shape=None,
+        min_axis=None,
+        max_axis=2,
+        force_int_axis=True,
+    ),
+    norm=st.sampled_from(["backward", "ortho", "forward"]),
+    n=st.integers(min_value=2, max_value=5),
+    s=st.lists(st.integers(), min_size=2, max_size=2).filter(lambda x: len(x) == 2),
```

Comment:
You don't need both `n` and `s` here, and since you have set `min_size=max_size=2` I don't think you need `filter`.